mod_form.php

```diff
@@ -182,32 +185,32 @@ public function definition() {
 
         // Users who pass the requirements but not have credential.
         if (isset($userswithcredential) && count($userswithcredential) > 0) {
-            $unissuedheader = false;
+            $mform->addElement('header', 'chooseunissuedusers', get_string('unissuedheader', 'accredible'));
+            $mform->addElement('html', '<div class="manual-issue-warning hidden">');
+            $mform->addElement('static', 'nouserswarning', '', get_string('nouserswarning', 'accredible'));
+            $mform->addElement('html', '</div>');
+            $mform->addElement('static', 'unissueddescription', '', get_string('unissueddescription', 'accredible'));
+            $this->add_checkbox_controller(2, 'Select All/None');
+            $mform->addElement('html', '<div id="unissued-users-container">');
 
             foreach ($userswithcredential as $user) {
-                if (accredible_check_if_cert_earned($accrediblecertificate, $course, $user) && !$user['credential_id']) {
-                    if (!$unissuedheader) {
-                        $mform->addElement('header', 'chooseunissuedusers', get_string('unissuedheader', 'accredible'));
-                        $mform->addElement('static', 'unissueddescription', '', get_string('unissueddescription', 'accredible'));
-                        $this->add_checkbox_controller(2, 'Select All/None');
-                        $unissuedheader = true;
-                    }
-
+                if (!$user['credential_id'] && accredible_check_if_cert_earned($accrediblecertificate, $user)) {
                     // No existing certificate, add this user to the unissued users list.
                     $mform->addElement('advcheckbox', 'unissuedusers['.$user['id'].']',
                         $user['name'] . '    ' . $user['email'], null, array('group' => 2));
                 }
             }
+            $mform->addElement('html', '</div>');
         }
 
         // Manually issue certificates header.
         $mform->addElement('header', 'chooseusers', get_string('manualheader', 'accredible'));
         // Hidden message to be displayed with Javascript when no users are available.
-        $mform->addElement('html', '<div id="users-warning" class="hidden">');
+        $mform->addElement('html', '<div class="manual-issue-warning hidden">');
         $mform->addElement('static', 'nouserswarning', '', get_string('nouserswarning', 'accredible'));
         $mform->addElement('html', '</div>');
-        $mform->addElement('html', '<div id="users-container">');
         $this->add_checkbox_controller(1, 'Select All/None');
```

Comment:
Minor - This is just a small thing and can be ignored but the state of `Select All/None` won't be reset when the selected group is changed. It means if the user clicks `Select All/None` to select all users and change the group, the user will have to click `Select All/None` twice to select all users again.